manifests/init.pp

```diff
@@ -33,6 +34,7 @@
   Boolean $manage_repo,
   Boolean $oss,
   Optional[String] $package_source,
+  String[1] $service_name,
```

Comment:
Please set the default value here instead of Hiera so that the default value is properly documented.

Reply:
Ok but i will do this modification when i come back after my holidays ;) perhaps i will add something to manage package name in same time

Reply:
Done and i add a new parameter 'package_name'